presto-product-tests/src/main/java/io/prestosql/tests/hive/acid/TestPartitionedFullAcidTable.java

```diff
@@ -0,0 +1,134 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive.acid;
+
+import com.google.inject.Inject;
+import io.prestosql.tempto.ProductTest;
+import io.prestosql.tempto.Requirement;
+import io.prestosql.tempto.RequirementsProvider;
+import io.prestosql.tempto.configuration.Configuration;
+import io.prestosql.tempto.context.ThreadLocalTestContextHolder;
+import io.prestosql.tempto.fulfillment.table.MutableTablesState;
+import io.prestosql.tempto.fulfillment.table.hive.HiveTableDefinition;
+import io.prestosql.tempto.query.QueryExecutor;
+import io.prestosql.tempto.query.QueryResult;
+import org.apache.thrift.TException;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+
+import static io.prestosql.tempto.Requirements.compose;
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.fulfillment.table.TableRequirements.mutableTable;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.HIVE_ACID;
+import static io.prestosql.tests.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.prestosql.tests.hive.acid.AcidTestHelper.simulateAbortedHiveTranscation;
+
+public class TestPartitionedFullAcidTable
+        extends ProductTest
+        implements RequirementsProvider
+{
+    private static String tableName = "ORC_acid_single_int_column_partitioned_fullacid";
+
+    private static final HiveTableDefinition SINGLE_INT_COLUMN_PARTITIONED_ORC = singleIntColumnPartitionedTableDefinition(Optional.empty());
+
+    private static HiveTableDefinition singleIntColumnPartitionedTableDefinition(Optional<String> serde)
+    {
+        return HiveTableDefinition.builder(tableName)
+                .setCreateTableDDLTemplate(buildSingleIntColumnPartitionedTableDDL(serde))
+                .setNoData()
+                .build();
+    }
+
+    private static String buildSingleIntColumnPartitionedTableDDL(Optional<String> rowFormat)
+    {
+        StringBuilder sb = new StringBuilder();
+        sb.append("CREATE %EXTERNAL% TABLE IF NOT EXISTS %NAME%(");
+        sb.append("   col INT,");
+        sb.append("   fcol INT");
+        sb.append(") ");
+        sb.append("PARTITIONED BY (part_col INT) ");
+        if (rowFormat.isPresent()) {
+            sb.append("ROW FORMAT ").append(rowFormat.get());
+        }
+        sb.append(" STORED AS ORC");
+        sb.append(" TBLPROPERTIES ('transactional'='true')");
+        return sb.toString();
+    }
+
+    @Inject
+    private MutableTablesState tablesState;
```

Comment:
Same comment as in https://github.com/prestosql/presto/pull/1034/files#r324635251

> I don't find this mechanism helpful. In fact, it would be easier to have a helper method which creates the table and use it in directly in every test.
> Bonus advantage is that you can easily reduce boiler plate by merging partitioned and unpartitioned tests together, by parameterizeding the table creation code.
>